src/android/CordovaAdalPlugin.java

```diff
@@ -231,7 +231,7 @@ private boolean deleteTokenCacheItem(String authority, boolean validateAuthority
             return true;
         }
 
-        String key = CacheKey.createCacheKey(itemAuthority, resource, clientId, isMultipleResourceRefreshToken, userId);
+        String key = CacheKey.createCacheKey(itemAuthority, resource, clientId, isMultipleResourceRefreshToken, userId, null);
```

Comment:
Hi, as far as I understood from reading the code, seems like if client id is available, it will delete items of this client id, even if family client id not passed.